src/assets/styles/style.less

```diff
@@ -2,13 +2,13 @@
 @import url(https://fonts.googleapis.com/css?family=Roboto:400,100,300,500);
 
 @import "variables.less";
-@import "logo.less";
-@import "sidebar.less";
-@import "search.less";
-@import "favourites.less";
-@import "star.less";
-@import "showcase-changes.less";
-@import "custom-scroll.less";
+@import "../../logo.less";
```

Comment:
The pattern is- keep the style in assets, unless it directly relates to a directive.